scripts/commands.js

```diff
@@ -2,8 +2,10 @@ require('dotenv').config()
 const { BOT_PREFIX } = process.env
 const Discord = require('discord.js')
 
+const changeEnv = require('../server.js')
+
 exports.study = async (message, args, guildRoles) => {
-    if(!typeof([args][0], 'int') && args) return 0
+    if(!typeof([args][0], 'int') && args) return message.reply(`To enter study mode:\n${BOT_PREFIX}study`)
```

Comment:
Why the return statement? It clutters the code. You don't use this return value and the other commands don't give a return value either, I would say make this 2 lines - message.reply and then the return